From c536c1ec159af1b5667d750f2da1fc912df7e950 Mon Sep 17 00:00:00 2001
From: Vihang Karajgaonkar <vihang@cloudera.com>
Date: Fri, 22 Jul 2016 10:49:36 -0700
Subject: [PATCH 0694/1164] CDH-42508 : Revert "CDH-42361 : HIVE-10624: Update
 the initial script to make beeline bucked cli as
 default and allow user choose old hive cli by env
 (Ferdinand via Xuefu)"

This reverts commit 4d6d084ab3cd2b575b8eeef87a0703aab961c815.

Change-Id: I46181c477a4c4ffd0df6fde9f51c688d16a78197
---
 bin/ext/cli.sh              |   11 ++++++-----
 bin/ext/util/execHiveCmd.sh |    6 ++----
 2 files changed, 8 insertions(+), 9 deletions(-)

diff --git a/bin/ext/cli.sh b/bin/ext/cli.sh
index ef0e8d0..ce26042 100644
--- a/bin/ext/cli.sh
+++ b/bin/ext/cli.sh
@@ -28,11 +28,12 @@ updateCli() {
 }
 
 cli () {
-  updateCli
-  execHiveCmd $CLASS $JAR "$@"
+  CLASS=org.apache.hadoop.hive.cli.CliDriver
+  execHiveCmd $CLASS "$@"
 }
 
 cli_help () {
-  updateCli
-  execHiveCmd $CLASS $JAR "--help"
-}
\ No newline at end of file
+  CLASS=org.apache.hadoop.hive.cli.CliDriver
+  execHiveCmd $CLASS "--help"
+} 
+
diff --git a/bin/ext/util/execHiveCmd.sh b/bin/ext/util/execHiveCmd.sh
index e46ec3c..167cc40 100644
--- a/bin/ext/util/execHiveCmd.sh
+++ b/bin/ext/util/execHiveCmd.sh
@@ -16,11 +16,9 @@
 execHiveCmd () {
   CLASS=$1;
   shift;
-  JAR=$1
-  shift;
 
   # cli specific code
-  if [ ! -f ${HIVE_LIB}/$JAR ]; then
+  if [ ! -f ${HIVE_LIB}/hive-cli-*.jar ]; then
     echo "Missing Hive CLI Jar"
     exit 3;
   fi
@@ -30,5 +28,5 @@ execHiveCmd () {
   fi
 
   # hadoop 20 or newer - skip the aux_jars option. picked up from hiveconf
-  exec $HADOOP jar ${HIVE_LIB}/$JAR $CLASS $HIVE_OPTS "$@"
+  exec $HADOOP jar ${HIVE_LIB}/hive-cli-*.jar $CLASS $HIVE_OPTS "$@"
 }
-- 
1.7.9.5

